.gitignore

```diff
@@ -21,3 +21,10 @@ db.php
 .Trashes
 ehthumbs.db
 Thumbs.db
+
+# Local PHPCS configuration files
+.phpcs.xml
+phpcs.xml
```

Comment:
Because, they are intended to be local-only files :-)

The distributable versions have the `.xml.dist` extension - what's in the repo.

The `.xml` versions are optional and local to each developer - they may want to change how or what PHPCS checks temporarily, and adding a `.xml` file is better than amending `.xml.dist` file which may then get accidentally committed.

It's standard practice, since PHPCS introduced support for `.xml.dist` extentions. [Here's WP doing the same](https://core.trac.wordpress.org/browser/trunk/.gitignore#L10). PHPUnit also supports `.xml.dist` and `.xml` files as well.